src/app/item-popup/Plug.m.scss

```diff
@@ -6,6 +6,11 @@
   height: var(--item-size);
   width: var(--item-size);
   border: none;
+
+  &:hover > circle:last-of-type {
```

Comment:
I'd probably prefer assigning a class directly to the circle in question instead of selecting it via last-of-type